dl1_data_handler/tests/test_reader.py

```diff
@@ -35,9 +35,55 @@ def test_r1_waveform_reading(r1_tmp_path, r1_gamma_file):
     )
     # Create an image reader and test basic properties
     r1_reader = DLWaveformReader(input_url_signal=[r1_gamma_file], config=config)
-    assert r1_reader._get_n_events() == 1
-    assert r1_reader.tel_type == "LST_LST_LSTCam"
+    assert r1_reader._get_n_events() == 1 # nosec
+    assert r1_reader.tel_type == "LST_LST_LSTCam" # nosec
     # Test the generation of a mono batch
     mono_batch = r1_reader.generate_mono_batch([0])
-    assert mono_batch["tel_id"] == 4
-    assert mono_batch["features"].shape == (1,  110, 110, 20)
\ No newline at end of file
+    assert mono_batch["tel_id"] == 4 # nosec
+    assert mono_batch["features"].shape == (1,  110, 110, 20) # nosec
+
+def test_dl1_hillas_parameter_extraction(dl1_gamma_file):
```

Comment:
Thanks a lot for adding the tests! But I think once you return all the parameters, you can have a simple check if the keys of your returning `dict` matches the column names. Please make this check in the existing tests, since there is no need of initializing the `DLImageReader` again.

Reply:
Thanks!